diff --git a/node_modules/vitest/dist/chunks/startModuleRunner.C2tTvmF9.js b/node_modules/vitest/dist/chunks/startModuleRunner.C2tTvmF9.js
index 25a42f9..36eaeba 100644
--- a/node_modules/vitest/dist/chunks/startModuleRunner.C2tTvmF9.js
+++ b/node_modules/vitest/dist/chunks/startModuleRunner.C2tTvmF9.js
@@ -437,8 +437,8 @@ class VitestTransport {
 		if (event.type !== "custom") return { error: /* @__PURE__ */ new Error(`Vitest Module Runner doesn't support Vite HMR events.`) };
 		if (event.event !== "vite:invoke") return { error: /* @__PURE__ */ new Error(`Vitest Module Runner doesn't support ${event.event} event.`) };
 		const { name, data } = event.data;
-		if (name !== "fetchModule") return { error: /* @__PURE__ */ new Error(`Unknown method: ${name}. Expected "fetchModule".`) };
 		if (name === "getBuiltins") return { result: [...nodeBuiltins, /^node:/] };
+		if (name !== "fetchModule") return { error: /* @__PURE__ */ new Error(`Unknown method: ${name}. Expected "fetchModule".`) };
 		try {
 			return { result: await this.options.fetchModule(...data) };
 		} catch (error) {
